src/main/java/jp/mincra/mincramagics/skill/combat/Beast_Spawn.java

```diff
@@ -0,0 +1,51 @@
+package jp.mincra.mincramagics.skill.combat;
+
+import jp.mincra.bktween.BKTween;
+import jp.mincra.bktween.TickTime;
+import jp.mincra.mincramagics.MincraMagics;
+import jp.mincra.mincramagics.player.MincraPlayer;
+import jp.mincra.mincramagics.skill.MagicSkill;
+import jp.mincra.mincramagics.skill.MaterialProperty;
+import net.kyori.adventure.text.Component;
+import org.bukkit.DyeColor;
+import org.bukkit.Particle;
+import org.bukkit.World;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Player;
+import org.bukkit.entity.Wolf;
+
+public class Beast_Spawn extends MagicSkill {
+    @Override
+    public boolean onTrigger(Player player, MaterialProperty property) {
+        // MP, Cooldown
+        MincraPlayer mPlayer = playerManager.getPlayer(player.getUniqueId());
+        if (!canTrigger(mPlayer, property)) return false;
+        consumeMp(mPlayer, property);
+        setCooldown(mPlayer, property);
```

Comment:
want: super class の onTrigger() を呼ぶようにお願いしますー (共通化のため)
```suggestion
        if (!super.onTrigger(player, property)) return false;
```